examples/task-scheduler-appointment/task_scheduler_appointment.bal

```diff
@@ -0,0 +1,48 @@
+import ballerina/io;
+import ballerina/task;
+import ballerina/runtime;
+
+int reminderCount = 0;
+
+public function main () {
+    // Appointment data record is used to provide appointment configurations.
+    task:AppointmentData appointmentData = {
+        seconds: "0/2",
+        minutes: "*",
+        hours: "*",
+        daysOfMonth: "?",
+        months: "*",
+        daysOfWeek: "*",
+        year: "*"
+    };
+
+    // Create an Appointment using the configurations.
+    task:Scheduler appointment = new({ appointmentDetails: appointmentData });
+
+    // Attach the service to the Scheduler.
+    _ = appointment.attach(appointmentService);
+
+    // Start the scheduler.
+    _ = appointment.start();
+
+    // Wait for sometime
```

Comment:
```suggestion
    // Wait for 10000 milliseconds.
```

Reply:
10 seconds should be better I guess. Will change.